host/src/CoreToolsHost/Program.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+
+using System.Text.Json;
+
+namespace CoreToolsHost
+{
+    internal class Program
+    {
+        static bool isVerbose = false;
+        static async Task Main(string[] args)
+        {
+            try
+            {
+                if (args.Contains("--verbose"))
```

Comment:
Can simply to ` isVerbose = args.Contains("--verbose");` (Move the string to a constant as well since it's being used in many places.)